examples/demo13.py

```diff
@@ -174,7 +174,7 @@ def main(argv):
         # Note that we could use wcs.toWorld() to get the (RA, dec) for these (x, y) positions.  Or,
         # if we had started with (RA, dec) positions, we could have used wcs.toImage() to get the
         # CCD coordinates for those positions.
-        mag_stamp.append(cat.param_cat.mag_auto[pos_rng()*cat.nobjects])
+        mag_stamp.append(cat.param_cat[pos_rng()*cat.nobjects][1])
```

Comment:
Don't use `[1]`.  You should always avoid magic numbers in code.  The better fix here is `param_cat['mag_auto'][...]`.

Also, I'd add an `int(...)` for the random number in the argument, since currently it's a float, which numpy must be converting to an `int` automatically, but it would be more intuitive to do that directly here, rather than rely on this kind of non-obvious feature.